.gitignore

```diff
@@ -1,4 +1,5 @@
 bower_components
 node_modules
 package-lock.json
-coverage
+yarn.lock
+coverage
```

Comment:
nit: 
![image](https://user-images.githubusercontent.com/6059356/42217485-1345e008-7ece-11e8-80f2-f4973769e977.png)

this red symbol means that there's no trailing newline in this file. 

To fix the and prevent such issues in future I recommend this VS Code setting: 

```json
  "files.insertFinalNewline": true,
```